torch_geometric/logging.py

```diff
@@ -25,7 +25,7 @@ def init_wandb(name: str, **kwargs):
 
 def log(**kwargs):
     def _map(value: Any) -> str:
-        if isinstance(value, int):
+        if isinstance(value, int) and not isinstance(value, bool):
```

Comment:
Not sure I understand this change.

Reply:
I think what happens when you pass in a boolean flag is it gets coerced into an int and logs as:

```python
from torch_geometric.logging import log

log(use_opt=True)
```

outputs:
```bash
use_opt: 001
```

with my change the output is:
```bash
use_opt: True
```

Let me know if you'd prefer to handle this in a separate patch or even drop it.

Reply:
Very possible this is my bad python, I tried:

```python
vals = [1, True, False, 0, '']

[isinstance(v, int) and not isinstance(v, bool) for v in vals]
```

outputs:
```
[True, False, False, True, False]
```

Removing the `not isinstance(v, bool)` from the condition:
```python
vals = [1, True, False, 0, '']

[isinstance(v, int) for v in vals]
```
outputs:
```
[True, True, True, True, False]
```

Reply:
Ah here is the answer:

> The [bool](https://docs.python.org/3/library/functions.html#bool) class is a subclass of [int](https://docs.python.org/3/library/functions.html#int)

from  https://docs.python.org/3/library/functions.html#bool

Reply:
Oh wow :D Thanks for clarifying :)